README.md

```diff
@@ -1,16 +1,216 @@
-# Title
+# Hashes of Hashes Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+1. Recognize vocabulary term: "hash of hashes"
+2. Recognize that "Hashes of Hashes" are trees
+3. Create an `Hash` of `Hash`es
+4. Read scalar data from an `Hash` of `Hash`es
+5. Modify scalar data in an `Hash` of `Hash`es
 
 ## Introduction
 
-## SWBAT 1
+Having worked with `Array`s of `Array`s and `Array`s of `Hash`es, you're going
+to be pleasantly surprised that most of the syntax for `Hash`es of `Hash`es
+(HoH) works like AoAs, but with `Hash` keys as the components of a coordinate.
+Unlike an AoA where coordinates help you find a position in a grid, coordinates
+in HoH can be stacked deeply so that you can tunnel deeply into a HoH.
 
-## SWBAT 2
+### Key Image: Relationship Tree
+
+To help you when reading the rest of this lesson, keep in mind that a HoH is
+like a family tree. Every person in a family tree has two chromosomal parents.
+
+We can render any given individual like so:
+
+```ruby
+{
+  name: "",
+  mother: {},
+  father: {}
+}
+```
+
+Notably, any `mother` value is, itself, a node that can have a `:mother` and
+`:father` value. And this is true for each of those values, and each of those
+values...and so on. You might see an infinite regress here. Because of the
+potential to go infinitely deep, we'll keep our HoHs only one or two
+generations wide.
+
+(image)
+
+### From Hash to Array of Hashes
+
+Recall that an`Hash` is like a table, but in code form. It is a way for your
+program to store pieces of data as elements in a collection. We call those
+elements, in a `Hash`, the `Hash`'s _values_. We can point to each _value_ by
+using a unique `String` or `Symbol` "lookup name" called a _key_. Arrays can
+contain any combination of data types -- `Boolean`s, `Integer`s, `Strings`.
+
```

Comment:
This seems like it may be missing something. There isn't much explanation regarding array of hashes. Maybe a visual example would be good. Something like An array of car candy with names and prices maybe?